tests/tpch/generate-data.py

```diff
@@ -117,7 +125,6 @@ def _tpch_data_gen(
             f"""
             SET memory_limit='{psutil.virtual_memory().available // 2**30 }G';
             SET preserve_insertion_order=false;
-            SET threads TO 1;
```

Comment:
Removing this upsets the target partition size, as the page size calculated to meet the partition size will be written out per thread.

Reply:
Thanks. Resolved.